weave/trace_server/trace_server_interface.py

```diff
@@ -45,7 +45,7 @@ class LLMCostSchema(LLMUsageSchema):
 
 class WeaveSummarySchema(ExtraKeysTypedDict, total=False):
     status: Optional[Literal["success", "error", "running"]]
-    nice_trace_name: Optional[str]
+    trace_name: Optional[str]
     latency: Optional[int]
```

Comment:
can you comment what fidelity latency is? It is an int, so i am hoping it is at least ms

Reply:
in the past, i have favored using a suffix like `latency_[UNIT]` to make this very clear